pkl-core/src/main/java/org/pkl/core/ast/type/ResolveDeclaredTypeNode.java

```diff
@@ -73,11 +74,36 @@ protected VmTyped getImport(
     var result = module.getCachedValue(importName);
     if (result == null) {
       result = callNode.call(member.getCallTarget(), module, module, importName);
+
+      var importedModule = (VmTyped) result;
+      if (importedModule.isNotInitialized()
+          && importedModule.isModuleObject()
+          && importedModule.getModuleInfo().isAmend()) {
+        // this is an amending module. Try to find the prototype
+        var prototypeModule = findPrototypeModule(importedModule);
+        if (prototypeModule != null) {
+          result = prototypeModule;
+        }
+      }
```

Comment:
The result of `getImport` should always be the prototype in all cases, even when the module has not yet been initialized.